api/src/main/java/ai/djl/modality/audio/Audio.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ * with the License. A copy of the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+ * OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+package ai.djl.modality.audio;
+
+import java.io.IOException;
+import java.io.OutputStream;
+
+/**
+ * {@code Audio} is a container of an audio in DJL. The raw data of the audio is wrapped in a float
+ * array.
+ */
+public interface Audio {
+
+    /**
+     * Get the float array data.
+     *
+     * @return The float array data.
+     */
+    float[] getData();
+
+    /**
+     * Get the number of channels of an audio file.
+     *
+     * @return The number of channels of an audio file.
+     */
+    int getChannels();
+
+    /**
+     * Save the audio to file.
+     *
+     * @param os {@link OutputStream} to save the audio.
+     * @param type type of the audio, such as "wav"
+     * @throws IOException audio cannot be saved through output stream
+     */
+    void save(OutputStream os, String type) throws IOException;
```

Comment:
Let remove this since we didn't have any implementation now

Reply:
Done